src/test/multi-dns-verified.spec.ts

```diff
@@ -2,7 +2,8 @@ import { location, navigateToVerify, uploadDocument, validateIframeTexts, valida
 
 fixture("Multiple DNS Verified for Certificate Rendering").page`${location}`;
 
-test("Sample document is rendered correctly when multiple dns is verfied", async () => {
+// eslint-disable-next-line jest/no-disabled-tests
+test.skip("Sample document is rendered correctly when multiple dns is verfied", async () => {
```

Comment:
considering to remove

Reply:
i think should remove, can maybe ask the rest of the devs.